exercises/concept/captains-log/.meta/design.md

```diff
@@ -19,3 +19,16 @@
 - `strings`: know how to use string formatting.
 - `numbers`: know how to apply basic mathematical operators.
 - `arrays`: know how to retrieve array elements by their index.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `essential`: If the student did not use the `nextInt` method in the `randomShipRegistryNumber` or `randomPlanetClass` methods, instruct the student to do so.
+- `essential`: If the student did not use the `nextDouble` method in the `randomStardate` method, instruct the student to do so.
```

Comment:
Given that a solution needs to pass all tests in order to receive feedback from the analyzer; do you have an example of a solution that does _not_ use the `nextInt` or `nextDouble` methods?

Reply:
My train of thought was similar as the secrets concept exercise, that its going to be pretty rare to see someone solving it with another methods, but if it happens the analyzer comment triggers

Reply:
Well my reason for asking was mainly because I want to avoid the same situation as in Log Levels, where feedback is given even though the solution needs no improvement.

I think the best way to avoid this is to narrow down the cases in which certain comments are triggered. 

So in this case we can choose to only leave the `Random.nextInt` comment if a solution uses `Random.next` for example. Which I doubt will happen often, and if it does it's probably more of an actionable comment instead of an essential one.

Reply:
Do you see someone using `Random.next` to solve the ones with `nextInt`? hmm maybe remove those comments and only left the `String.format` one

Reply:
So far no, I have not seen any. I posted other findings below.

Reply:
I removed those comments finally